app/src/components/market/common/history_chart/index.tsx

```diff
@@ -1,163 +1,172 @@
+  const scalarHighNumber = scalarHigh && Number(formatBigNumber(scalarHigh, 18))
 
-  const mapPeriod: { [period in Period]: { totalDataPoints: number; blocksPerPeriod: number } } = {
-    All: { totalDataPoints: allDataPoints, blocksPerPeriod: blocksPerAllTimePeriod },
-    '1Y': { totalDataPoints: 365, blocksPerPeriod: blocksPerDay },
-    '1M': { totalDataPoints: 30, blocksPerPeriod: blocksPerDay },
-    '1W': { totalDataPoints: 7, blocksPerPeriod: blocksPerDay },
-    '1D': { totalDataPoints: 24, blocksPerPeriod: blocksPerHour },
-    '1H': { totalDataPoints: 60, blocksPerPeriod: blocksPerMinute },
+  const toScaleValue = (decimal: number, fixed = 0) => {
+    return `${(decimal * ((scalarHighNumber || 0) - (scalarLowNumber || 0)) + (scalarLowNumber || 0)).toFixed(
+      fixed,
+    )} ${unit}`
   }
 
-  useEffect(() => {
-    library.getBlockNumber().then((latest: number) => setLatestBlockNumber(latest - blocksOffset))
-  }, [blocksOffset, library])
-
-  useEffect(() => {
-    const getBlocks = async (latestBlockNumber: number) => {
-      const { blocksPerPeriod, totalDataPoints } = mapPeriod[period]
-
-      if (latestBlockNumber) {
-        const blockNumbers = range(totalDataPoints).map(multiplier => latestBlockNumber - multiplier * blocksPerPeriod)
-        const blocks = await Promise.all(blockNumbers.map(blockNumber => library.getBlock(blockNumber)))
-        setBlocks(blocks.filter(block => block))
-      }
-    }
-
-    if (latestBlockNumber) {
-      getBlocks(latestBlockNumber)
-    }
-    // eslint-disable-next-line
-  }, [latestBlockNumber, library, period])
-
-  const isScalar = isScalarMarket(oracle || '', context.networkId || 0)
-
-  return hidden ? null : (
-    <HistoryChart
-      holdingSeries={holdingsSeries}
-      isScalar={isScalar}
-      onChange={setPeriod}
-      options={keys(mapPeriod)}
-      outcomes={outcomes}
-      scalarHigh={scalarHigh}
-      scalarLow={scalarLow}
-      unit={unit}
-      value={period}
-    />
+  const renderScalarTooltipContent = (o: any) => {
+    const { label, payload } = o
+    const prediction = (
+      payload[0]?.value * ((scalarHighNumber || 0) - (scalarLowNumber || 0)) +
+      (scalarLowNumber || 0)
+    ).toFixed(2)
+    return (
+      <ChartTooltip>
+        <TooltipTitle>{label}</TooltipTitle>
+        <Legends>
+          <Legend key={`item-0`}>
+            <AnEvenSmallerLittleBall outcomeIndex={0} />
+            <strong>{`${prediction}`}</strong>
+            {`${unit}`}
+          </Legend>
+        </Legends>
+      </ChartTooltip>
+    )
+  }
+  return (
+    <>
+      <ResponsiveWrapper>
+        <ResponsiveContainer height={300} width="100%">
+          <AreaChart data={data} margin={{ top: 0, right: 0, left: 0, bottom: 0 }} stackOffset="expand">
+            <XAxis dataKey="date" stroke="#E8EAF6" tick={{ fill: '#757575', fontFamily: 'Roboto' }} tickMargin={11} />
+            <YAxis
+              orientation="right"
+              stroke="#E8EAF6"
```

Comment:
Can we set these colors in themes and import from there?
I imagine there might be multiple charts in the future that would want to have the same look and feel

Reply:
well i would need to import theme in component and then use it like that since i don't have access to themes in this scope should i do that?

Reply:
![Screenshot 2021-01-13 at 23 12 14](https://user-images.githubusercontent.com/33226956/104516579-dc387200-55f4-11eb-9621-5a9c6d402104.png)
Is this good? I haven't seen that we use themes like this but does work....